package.json

```diff
@@ -8,23 +8,29 @@
     "test": "mocha"
   },
   "dependencies": {
-    "async": "^0.9.0",
+    "async": "^0.9.2",
     "debug": "^2.1.1",
     "loopback-connector": "^2.1.0",
     "mysql": "^2.5.4",
-    "strong-globalize": "^2.5.8"
+    "strong-globalize": "^2.6.6"
   },
   "devDependencies": {
-    "bluebird": "~2.9.10",
-    "loopback-datasource-juggler": "^2.28.0",
-    "mocha": "^2.1.0",
+    "bluebird": "~2.9.34",
+    "eslint": "^2.13.1",
+    "eslint-config-loopback": "^4.0.0",
+    "loopback-datasource-juggler": "^2.48.0",
+    "mocha": "^2.5.3",
     "rc": "^1.0.0",
-    "should": "^8.0.2",
+    "should": "^8.4.0",
     "sinon": "^1.15.4"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/strongloop/loopback-connector-mysql.git"
   },
+    "scripts": {
```

Comment:
I missed this one, @loay Please leave in the pretest.js script, why are you removing it?